docs/docs/technical/logger.md

```diff
@@ -0,0 +1,179 @@
+# Logging Server
+
+The PatternPal extension offers valuable research opportunities, thus
+requiring the implementation of an adequate logging solution. The
+logging system operates in conjunction with PatternPal's service module
+to process source code, recognition scores and other relevant data for a
+specific snapshot appropriately.
+
+## System requirements
+
+### Hardware requirements (minimum)
+
+- Processor: Dual-core processor
+
+- Memory: 4GB RAM
+
+- Storage: 32GB
+
+### Software requirements
+
+- Operating system: Linux distribution (Ubuntu, CentOS, or Debian)
+
+- Container manager: Latest stable version of Docker
+
+### Docker images
+
+- .NET SDK image
+
+- ASP.NET Core Runtime image
+
+### Other
+
+- Remote database server: PostgreSQL; A remote PostgreSQL database
+server. This can be a cloud-based, managed database service, or a
+dedicated server.
+
+## Installation
+
+1. Install Linux: Install a compatible Linux distribution on the target
+server. Follow the installation instructions of the chosen Linux
+Distribution.
+
+2. Install Docker: Install Docker on the Linux server. Follow the
+docker installation instructions for the installed OS
+[[here](https://docs.docker.com/desktop/install/linux-install/)]{style="color: blue"}.
+
+3. Pull Docker Image: Pull the Docker image for the application from
+the docker repository or Docker Hub. The current Docker repository
+is 'siem2l/patternpalloggingserver:latest'
+
+4. Run Docker Container: Create and run the pulled docker container
+from step 3.
+
+5. Setup Database: Set up a remote PostgreSQL database server,
+Following the instruction provided by your chosen database service
+or server.
+
+6. Configuration: Update the application settings or configuration
+files to connect to the database using the correct connection
+string. This can be done using volume-mounted files.
```

Comment:
I found "this can be done using volume-mounted files" a bit confusing while setting up the server myself. Maybe we can improve this?

Reply:
Removed this entire sentence this is just confusing haha